kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -333,6 +388,19 @@ protected static void ensureValidMap(String name, List<String> values) {
     );
   }
 
+  /**
+   * Return a Map detailing which BigQuery table each topic should write to.
+   * @return A Map associating Kafka topic names to BigQuery table names.
+   */
+  public Map<String, String> getTopicsToTables() {
```

Comment:
instead of having two methods, onf ot which returns topics -> tables and the other which returns topics -> datasets, could we have a single method that returns topics to TableIds? (which includes dataset and table info)